resources/HookTypes.ts

```diff
@@ -317,8 +316,7 @@ export interface Card {
 }
 
 export interface TypedRequestBody extends Express.Request {
-  body: Hook;
+  body: any;
```

Comment:
why change this to any? Is this meant to be a temporary change?

Reply:
it was causing issues with the intermediary. Then it was causing issues with REMS Admin when I fixed that. Keeping it as any gets it to work with everything. Its an issue with the type inheritance not the functionality itself